game-core/src/main/java/games/strategy/triplea/ai/flowfield/FlowFieldAI.java

```diff
@@ -0,0 +1,47 @@
+package games.strategy.triplea.ai.flowfield;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.GamePlayer;
+import games.strategy.engine.framework.startup.ui.PlayerType;
+import games.strategy.triplea.ai.AbstractAi;
+import games.strategy.triplea.delegate.remote.IAbstractPlaceDelegate;
+import games.strategy.triplea.delegate.remote.IMoveDelegate;
+import games.strategy.triplea.delegate.remote.IPurchaseDelegate;
+import games.strategy.triplea.delegate.remote.ITechDelegate;
+
+public class FlowFieldAI extends AbstractAi {
+  public FlowFieldAI(final String name) {
+    super(name);
+  }
+
+  @Override
+  public PlayerType getPlayerType() {
+    return null;
```

Comment:
side-note, an override that returns null is a code smell. Indicates an inappropriate abstraction where either there are really two interfaces or something.. Keeping one interface probably make sense, in which case favoring optional return types is preferred compared to nullable return types

Reply:
I missed that I needed to set this value.  It shouldn't return null.